script/utils.js

```diff
@@ -1,28 +1,34 @@
+
+const randomChars = 'abcdefghijklmnopqrstuvwxyz      '
+
+export function random (num) {
+
+    return Math.floor(Math.random() * num)
+
+}
+
 export function newLetters(length) {
 
-    let randomChars = 'abcdefghijklmnopqrstuvwxyz      '
-    let randomLetters = ''
-    
+    let randomLetters = []
```

Comment:
you can just push into an array instead of building a string and then splitting it